portfolio/src/main/webapp/script.js

```diff
@@ -215,7 +220,16 @@ function reset() {
     }
 }
 
-//Create a 16 by 16 grid
-const gridWidth = 40;
-const gridHeight = 40;
-populateGrid(gridWidth, gridHeight);
\ No newline at end of file
+function getGreeting() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
```

Comment:
When chaining promises like this, it's nice to have them on newlines for readability. eg
```
fetch('/data')
.then(response => response.text())
.then((greeting) => {
```

Reply:
Got it, I'll change that.